src/Template/Vocabulary/add_sentences.ctp

```diff
@@ -27,10 +27,15 @@
 ?>
 <?php
 $this->Html->script('/js/vocabulary/add-sentences.ctrl.js', ['block' => 'scriptBottom']);
-
-$title = __('Vocabulary that needs sentences');
-
-$this->set('title_for_layout', $this->Pages->formatTitle($title));
+    if (empty($langFilter)) {
+        $title = __('Vocabulary that needs sentences');
+        $this->set('title_for_layout', $this->Pages->formatTitle($title));
```

Comment:
This line should be moved after the if-block. Right now the page title is only set when there is no language filter.

And I suggest to remove the indentation for the whole if-block.

Reply:
nice catch!